### ISSUE TITLE
React warning for <option selected> inside toolbar dropdown

### ISSUE THREAD

**Issue by Janekk** [zenoamaro/react-quill#75]
Hi,

I noticed such console warning:

`Warning: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.`

coming from: 

```
<select title="Font" class="ql-font" style="display: none;">
  <option selected="" value="sans-serif">Sans Serif</option>
  <option value="serif">Serif</option>
  <option value="monospace">Monospace</option>
</select>
```

It's about how React prefers to handle `<select/>` elements

**Comment by zhang-z** [comment]
@Janekk , it has been fixed in pull request #73. You can install from master branch for this issue

**Comment by Janekk** [comment]
Thanks @zhang-z! Closing this one.
